### PR TITLE
[MonologBridge] Added WebSubscriberProcessor to ease processor configuration

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT
| Doc PR        |

---

Side note: I could be nice to have an interface to act as the marker to AutoConfigure processors

### CODE REVIEW

File: src/Symfony/Bridge/Monolog/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.1.0
+-----
+
+ * Added `WebSubscriberProcessor` to be easily autoconfigured
```
Comment (fabpot) [inline src/Symfony/Bridge/Monolog/CHANGELOG.md]: I don't understand the meaning of this sentence. What is easily configured?

File: src/Symfony/Bridge/Monolog/Processor/WebSubscriberProcessor.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+if (!interface_exists(EventSubscriberInterface::class)) {
+    throw new \RuntimeException('The "WebSubscriberProcessor" requires the EventDispatcher component. Please run "composer require symfony/event-dispatcher".');
+}
+
+/**
+ * WebProcessor override to read from the HttpFoundation's Request and to be autoconfigured.
```
Comment (fabpot) [inline src/Symfony/Bridge/Monolog/Processor/WebSubscriberProcessor.php]: Sorry, but I don't understand what that means as well (I see that you copy/pasted the one from WebProcessor, but the second part is quite cryptic to me).

**Review by fabpot — CHANGES_REQUESTED** [review]

**Review by fabpot — APPROVED** [review]
minor CS comment

**Comment by lyrixx** [comment]
> minor CS comment

@fabpot I don't see your comments :/

File: src/Symfony/Bridge/Monolog/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.1.0
+-----
+
+ * `WebProcessor` now implements EventSubscriberInterface in order to be easily autoconfigured
```
Comment (fabpot) [inline src/Symfony/Bridge/Monolog/CHANGELOG.md]: missing quotes around EventSubscriberInterface

Comment (lyrixx) [reply src/Symfony/Bridge/Monolog/CHANGELOG.md]: Fixed

**Comment by fabpot** [comment]
Thank you @lyrixx.

**Pull request merged**